### DISCUSSION TITLE
在群晖的docker上部署完成登录成功后 重新启动容器 显示数据库连接失败

### DISCUSSION THREAD

**Discussion by JinShichang** [MoeNetwork/Tieba-Cloud-Sign#127] [疑问]
第一次启动完成后 进入web界面登录并绑定百度账号，一切功能正常，然后关掉容器再启动，再次进入web界面 显示连接数据库失败，数据库用户名或密码错误，数据库并没有任何更改，必须删掉容器再次重新配置才能使用 ，停止再启动问题依旧

**Comment by 7161457** [comment]
你好 请问你是怎么解决的，我也是这个问题。

**Reply by JinShichang** [reply]
其实就是数据库没有赶在贴吧签到前启动起来，导致连不上，解决方法也简单，先等数据库的容器完全起来后，再启动贴吧签到，把贴吧签到的开机自启关了，用命令行设置一个延迟十分钟后再启动

**Reply by 7161457** [reply]
我的问题和你不一样，我是docker 反复重启都连接不上出现2002，数据库用phpmyadmin连接是正常的。

**Reply by JinShichang** [reply]
删了重搞吧

**Comment by silence9527** [comment]
你好，请问我是在群晖的docker里直接安装这个，还需要安装mysql之类的吗，我看检测上有mysqli通过，现在我安装完最后一直提示2002错误    #229 ，劳驾帮忙看看，谢谢
